src/HotChocolate/Core/src/Types.Scalars/ScalarResources.resx

```diff
@@ -249,4 +306,17 @@
   <data name="UnsignedIntType_IsNotUnsigned_ParseValue" xml:space="preserve">
     <value>UnsignedIntType cannot parse the provided value. The provided value is not greater than or equal to 0.</value>
   </data>
-</root>
+  <data name="IPv4Type_Description" xml:space="preserve">
+    <value>The IPv4 scalar type represents a valid a IPv4 address as defined
+ here https://en.wikipedia.org/wiki/IPv4.</value>
+  </data>
+  <data name="IPv4Type_IsInvalid_ParseLiteral" xml:space="preserve">
+    <value>IPv4Type cannot parse the provided literal. The provided value is not a valid IPv4 address.</value>
+  </data>
+  <data name="IPv4Type_IsInvalid_ParseValue" xml:space="preserve">
+    <value>IPv4Type cannot parse the provided value. The provided value is not a valid IPv4 address.</value>
+  </data>
+  <data name="IPv4Type_ValidationPattern" xml:space="preserve">
+    <value>(^(?:(?:(?:0?0?[0-9]|0?[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}(?:0?0?[0-9]|0?[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])(?:\/(?:[0-9]|[1-2][0-9]|3[0-2]))?)$)</value>
```

Comment:
This needs to be a constant.

Reply:
@michaelstaib  All the regular expressions used by the scalars are in Resources so that
- We do not have to worry about string escaping
- We do not have to split strings because of the 100 line limit

Reply:
Had a discussion on this with Pascal, and for consistency, all Regex are put in the resource files. 
@PascalSenn?